### DISCUSSION TITLE
Unable to detect some devices on Windows but I can on Mac

### DISCUSSION THREAD

**Discussion by MaxLutton** [hbldh/bleak#605] [Q&A]
Windows version: 10.0.162299
Python version: 3.9.6
bleak version: 0.12.1

I have some embedded BLE devices that I need to test, but my Windows machine isn't seemingly able to detect their advertisements with the BleakScanner or even with the Bluetooth LE Explorer... 

Any ideas why this would work on Mac, but not on Windows? Ideally, I'd like to programmatically pair with the devices, but at this point I can't even detect them.

**Comment by dlech** [comment]
Have you tried sniffing the Bluetooth packets? https://bleak.readthedocs.io/en/latest/troubleshooting.html#capture-bluetooth-traffic

**Reply by MaxLutton** [reply]
Thanks for the suggestion @dlech! From the packet capture, I'm able to see the advertisements from my device and a lot of advertisement data. However, each is classified as a "Malformed packet: BT Common". I do see that my device's packets are missing an RSSI value, but not sure what else could be afoot.
